src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -1244,6 +1245,12 @@ class debug_commandscript : public CommandScript
         handler->PSendSysMessage("Waypoint SQL written to SQL Developer log");
         return true;
     }
+
+    static bool HandleDebugDummyCommand(ChatHandler* handler, uint32 /*args*/)
```

Comment:
why need uint32?

Reply:
I didnt know what to put for args other than uint32

Reply:
Then make this command without arguments

Reply:
@Kitzunu